schemas/context/profile-person-details.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-person-details",
```

Comment:
@kstreeter I am open to changing this to person as you mentioned when we talked.  Because we currently have a complex type of that name `"https://ns.adobe.com/xdm/context/person"` it would need to be person-extension. Is that a pattern you want really? Does it actually help?